app/assets/javascripts/users.js.erb

```diff
@@ -220,9 +225,22 @@ function deleteTargetEvents() {
   });
 }
 
+function editFirstLoginEvent() {
+  $('.welcome-bottom .principal-button').click(function() {
+    var route = $('.principal-button').data('edit-first-login-path');
+    $.post(route).done( function() {
+      location.reload();
```

Comment:
Try to make it work without reloading, that way the behaviour is consistent with the other actions